src/lambda/commands/deleteCloudFormation.ts

```diff
@@ -10,27 +10,29 @@ import * as vscode from 'vscode'
 import { CloudFormationClient } from '../../shared/clients/cloudFormationClient'
 import { ext } from '../../shared/extensionGlobals'
 import { getLogger, Logger } from '../../shared/logger'
+import { recordCloudformationDelete, Result } from '../../shared/telemetry/telemetry'
 import { CloudFormationStackNode } from '../explorer/cloudFormationNodes'
 
 export async function deleteCloudFormation(refresh: () => void, node?: CloudFormationStackNode) {
     const logger: Logger = getLogger()
-    if (!node) {
-        vscode.window.showErrorMessage(
-            localize(
-                'AWS.message.error.cloudFormation.unsupported',
-                'Unable to delete a CloudFormation Stack. No stack provided.'
+    let deleteResult: Result = 'Succeeded'
+    const stackName = node?.stackName ?? ''
```

Comment:
I haven't seen the `??` operator before but I like using this here. May need to evaluate using this elsewhere in place of `||`

Reply:
Yep, thanks typescript 3.7